### PR TITLE
feat: Toolbox - top bar updates

### PR DESCRIPTION
**Pull request by tbwiss**
# Description

> **Note**
> 
> Please provide a description of the work completed in this PR below

Updating the Top bar to follow the latest design. This PR includes:

 - Dark top bar styling including logo and buttons
 - Add a new "context menu" to select the current database and allow the user to disconnect from the dbms
 - Tooltip with connection information on hover
 - Screen size: When tablet width (ca 1024px), alignment of buttons and visible information
 - Remove the 'beta' tag (since we do not release a new Toolbox version for now, this should be fine. Can be re-introduced if needed)
 - E2E tests to check the new functionalities

## Complexity

> **Note**
>
> Please provide an estimated complexity of this PR of either Low, Medium or High

Medium

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Made some changes to the Neo4j GraphQL Toolbox? Or made some changes to the core library perhaps?

Check out your changes at https://neo4j-graphql-toolbox-3412.surge.sh

Use for instance a local dbms hosted on [Neo4j Desktop](https://neo4j.com/download/) or a cloud-hosted [Aura Free](https://neo4j.com/cloud/aura-free/) dbms.

**Review by mjfwebb — APPROVED** [review]
Great work!

**Review by mjfwebb — DISMISSED** [review]
Great work!

File: packages/graphql-toolbox/tests/urlqueryparameters.test.ts [hunk 1]
```diff
@@ -101,4 +101,26 @@ base.test.describe("URL query parameters", () => {
...
+        page,
+        topBarPage,
+    }) => {
+        await page.goto(`/?db=${databaseName}`);
+
+        const login = new Login(page);
+        await login.login();
+
+        // We need some waiting time after loading the application for a dbms query checking for available databases to resolve.
+        await page.waitForTimeout(2000);
```
[the remark this answers is not in the document]
Comment (tbwiss) [reply packages/graphql-toolbox/tests/urlqueryparameters.test.ts]: Good one! There is indeed, https://github.com/neo4j/graphql/pull/3412/commits/3e625f22b4c39d850d24da5e8f403230fb3e9bdd

**Pull request merged**